kinetic/distribution.yaml

```diff
@@ -13875,6 +13875,16 @@ repositories:
       url: https://github.com/ethz-asl/ethzasl_xsens_driver.git
       version: master
     status: maintained
+  xsens_official:
```

Comment:
For consistency please have the name match the repository name so it's easier to find and match.